client/src/components/login.component.js

```diff
@@ -1,6 +1,9 @@
 import React, { Component } from "react";
+import { useHistory } from "react-router";
+import { Redirect } from 'react-router'
```

Comment:
You should be able to add Redirect by useHistory on the same line

Reply:
removed as not used on this page